config/scgitx.yml

```diff
@@ -21,4 +21,8 @@ base_branch: master
 staging_branch: staging
 prototype_branch: prototype
 
+aggregate_branches:
+  - prototype
+  - staging
```

Comment:
They're not *immutably* aggregate branches since you can modify the `staging_branch` and `prototype_branch` properties to override the defaults. I have a test case where this list is modified to include 'staging' and 'qa' in the config.